lib/utils.js

```diff
@@ -122,8 +123,16 @@ exports.isGreaterThanVersion = function isGreaterThanVersion(options) {
     let greaterVersion = options.greaterVersion;
     let smallerVersion = options.smallerVersion;
 
-    greaterVersion = Number(greaterVersion.toString().replace(/\./g, ''));
-    smallerVersion = Number(smallerVersion.toString().replace(/\./g, ''));
+    // Are they semver like strings?
+    const periodCheckRegex = RegExp(/\./g);
```

Comment:
Probably not required for `RegExp`, because it returns an instance anyway, but for consistency. `new` is missing.

Reply:
Oh yes you're right, it's a force of habit. I'll get that changed and pushed asap, thanks for the review 😄